lang/gnu-apl/files/gnu-apl

```diff
@@ -0,0 +1 @@
+bin/gnu-apl
```

Comment:
What is the function of this file?

Reply:
Do you mean I should add comments to the file?

Reply:
Like `bin/gnu-apl # this is the executable for APL interpreter`?

Reply:
But it isn't. It's just a one line file named "gnu-apl" with the words "bin/gnu-apl" in it.

Reply:
Ah, you are asking the function of files/gnu-apl. I thought you are asking the function of /bin/gnu-apl.

This file works with `apl_select`, so `port select --set apl gnu-apl` can symlink `gnu-apl` to `apl`. Also `apl_select/files/base` and `apl_select/files/none`. The files in `base` lays out the destination files and `gnu-apl` lists the source. And `-` in `none` means don't symlink. 

That's how I understand it and not guaranteed to be true. There is no official documentation at all and I asked on IRC and copied from other portfiles.

Reply:
Does it seem to work correctly?

Reply:
Yeah. Selecting `gnu-apl` and `none` work.